interactive/interactive_client.go

```diff
@@ -215,6 +234,15 @@ func (c *InteractiveClient) runInteractivePrompt(ctx context.Context) (ret utils
 			}
 			return
 		}),
+		prompt.OptionHighlighter(func(d prompt.Document) ([]byte, error) {
+			iterator, err := c.highlights.lexer.Tokenise(nil, d.Text)
+			if err != nil {
+				return nil, err
+			}
+			buffer := bytes.NewBuffer([]byte{})
+			c.highlights.formatter.Format(buffer, c.highlights.style, iterator)
```

Comment:
why do we need to pass style into format - if the highlighter already has a style property?